handbook/governance/code-of-ethics.md

```diff
@@ -0,0 +1,43 @@
+# dOrg Code of Ethics
+
+To ensure that dOrg remains an exceptional place to work and a valuable service provider to our clients & partners, we must hold ourselves to high ethical standards such as those outlined here.
+
+## Avoided Behaviors
+The following behaviors should be avoided:
+
+* Accepting work from dOrg clients/partners independently without paying a referral fee (20%) to dOrg for sourcing said work
+* Deletion or malicious alteration of communications/comments/messages on dOrg channels except, in cases, where ethical violations have occurred and such action has been approved by dOrg moderators
+* Engagement in abuse, discrimination, disrespect, harassment, or promotion of said or otherwise offensive content against/to any prospective/past/current dOrg client/partner/builder
+* Exertion of personal copyright over any work prepared for, paid for by, or otherwise commissioned by dOrg, or any past/current dOrg client/partner
+* Obstruction of dOrg operations through 'down staking' of DAO proposals except, in cases, where the proposer has admitted that a wish to rescind their proposal and/or a significant enough margin of voters have expressed a wish to change their votes such that the outcome (passing or failing) of the proposal would be otherwise changed
+* Participation in any strike, slowdown, sabotage, or work stoppage in retaliation for the action(s) or inaction of any prospective/past/current dOrg client/partner/builder
+* Public defamation of any prospective/past/current dOrg client/partner/builder without prior authorization and/or through unofficial channels
+
```

Comment:
I think we should add a line clarifying the issue of privacy / public by default. Maybe something to the effect of "Sharing documents, code, media, or other records explicitly marked as internal, confidential, or private outside of dOrg. Or sharing any materials related to client/partner projects without prior authorization by the client/partner."

Reply:
I added this